package.json

```diff
@@ -32,18 +32,17 @@
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jsdoc": "^39.2.9",
     "husky": "^4.2.5",
-    "jest": "^27.3.1",
+    "jest": "^29.5.0",
     "lint-staged": "^10.2.11",
     "prettier": "^2.0.5",
     "rimraf": "^2.6.2",
-    "ts-jest": "^27.0.7",
+    "ts-jest": "^29.1.0",
     "ts-node": "^10.7.0",
-    "typescript": "^4.5.2"
+    "typescript": "^5.0.3"
```

Comment:
need to update `typescript` because `get-tsconfig` uses the new `.d.cts` and `.d.mts` file extensions which were not supported by TypeScript `v4.5.2`